code/datums/supply_packs/medical.dm

```diff
@@ -1,51 +1,3 @@
-//*******************************************************************************
-//MEDICAL
-//*******************************************************************************/
-
-
-/datum/supply_packs/medical
```

Comment:
this one has legit usecases, marines often ask for medical supplies to be dropped. it's more efficient to get belts from medical of course buuuuuuuuut...

Reply:
didn't know that never saw it being used i can leave it...